documentation/requirements.txt

```diff
@@ -7,3 +7,4 @@ sphinxcontrib-makedomain
 sphinxcontrib-spelling
 pyenchant
 sphinx-issues
+towncrier
```

Comment:
Actually, only the person doing a release needs to have towncrier, but it probably doesn't hurt listing it here anyway?

Reply:
Ideally we'd run towncrier as part of building the docs to get "in development" release notes, but we can do that in a future PR.

Reply:
Will do that in this PR.